bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -310,6 +310,19 @@ spec:
                         items:
                           type: string
                         type: array
+                      logLevel:
+                        description: Velero server’s log level (default info, use
+                          debug for the most logging)
+                        enum:
```

Comment:
`warn` and `trace` log levels are not supported by Velero https://github.com/vmware-tanzu/velero/blob/master/design/cli-install-changes.md

Reply:
[Trace](https://github.com/vmware-tanzu/velero/blob/9f0ea22c608dfa04f44e8ffdd7516d13a34ba0d9/pkg/plugin/clientmgmt/logrus_adapter.go#L179) is supported.
[Warn](https://github.com/vmware-tanzu/velero/blob/9f0ea22c608dfa04f44e8ffdd7516d13a34ba0d9/pkg/plugin/clientmgmt/logrus_adapter.go#L185) is supported.
I'll update docs upstream.

Reply:
Also pretty sure [main](https://github.com/vmware-tanzu/velero/blob/main/design/cli-install-changes.md) branch is default in velero now

Reply:
File an issue rather.. seems inconsistency need to be resolved.

Reply:
nevermind.. [design/cli-install-changes.md](https://github.com/vmware-tanzu/velero/blob/master/design/cli-install-changes.md) is a design doc.. not usage docs.

Reply:
![image](https://user-images.githubusercontent.com/11228024/157325506-d84b2b32-9c56-4c54-bcea-94d68334389e.png)
No mention of supported levels in markdown. But the [code](https://github.com/vmware-tanzu/velero/blob/9f0ea22c608dfa04f44e8ffdd7516d13a34ba0d9/pkg/plugin/clientmgmt/logrus_adapter.go#L176-L189) clearly states that trace, debug, info, warn, and error are supported.